### DISCUSSION TITLE
Can I use Gutenberg Block editor my Reactjs Project?

### DISCUSSION THREAD

**Discussion by MicrotersAsad** [WordPress/gutenberg#66395] [General]
**Can I use Gutenberg Block editor my Reactjs Project?**

**Comment by t-hamano** [comment]
Hi @MicrotersAsad, Have you checked out [this documentation](https://github.com/WordPress/gutenberg/blob/trunk/platform-docs/docs/intro.md)? It's the documentation for building your own block editor.

If you have already cloned the Gutenberg repository, you can also start a local documentation site with the following command:

```
cd ./platform-docs
npm install
npm run start
```

**Reply by MicrotersAsad** [reply]
Do you have a video tutorial on how to set up Gutenberg in my React project? I need to replicate all the functionalities of Gutenberg, just like it works in WordPress

**Reply by MicrotersAsad** [reply]
I tried setting it up, but it's not working correctly because the settings options, preview option, color selection, and other features aren't showing up.

**Comment by t-hamano** [comment]
> Do you have a video tutorial on how to set up Gutenberg in my React project?

As far as I know there are no video tutorials yet.

> it's not working correctly because the settings options, preview option, color selection, and other features aren't showing up.

If you want to build those features outside of WordPress, but in the same way as WordPress, I think you would have to implement them yourself via various APIs and components.
